packages/mds-policy/api.ts

```diff
@@ -133,15 +133,19 @@ function api(app: express.Express): express.Express {
   })
 
   app.get(pathsFor('/policies'), (req, res) => {
-    // TODO extract start/end applicability
-    // TODO filter by start/end applicability
-    const { start_date = now(), end_date = now() } = req.query
+    const { start_date = now(), end_date = now(), unpublished } = req.query
     log.info('read /policies', req.query, start_date, end_date)
     if (start_date > end_date) {
       res.status(400).send({ result: 'start_date after end_date' })
       return
     }
-    db.readPolicies({ start_date, end_date })
+
+    let get_unpublished = false
+    if (unpublished !== undefined || unpublished !== null) {
```

Comment:
I don't think `unpublished` should be something that's part of the query... `unpublished` policies should be viewable to a specific audience (i.e. policy authors). I think this makes more sense to have as an entirely separate endpoint, as this change would differ from the current Policy spec for the `/policies` endpoint.

Reply:
And it's important that unpublished policies are only viewable by the author as an audience.

Reply:
I mean, it's in the draft API spec. "The only addition is that when calling the GET /policies endpoint, append ?unpublished to get the unpublished ones, separate from the published ones.  Adding ?unpublished will have no effect if your access token has a provider_id.
"
@karcass what's the motivation for this query param?